auto-discovery-templates/autogen-sensors.py

```diff
@@ -26,6 +26,18 @@
     "on_time_total_kwh"
 ]
 
+
+topics_units = {
+    "sys_wifi_strength": "dBm",
+    "on_time_30_days": "s",
+    "on_time_7_days": "s",
+    "on_time_total": "s",
+    "on_time_30_days_kwh":"kwh",
```

Comment:
HA expects the unit to be `kWh` otherwise they won't be visible in the Energy section.